dcs/liveries/livery.py

```diff
@@ -86,13 +86,16 @@ def from_lua(code: str, path: str) -> Optional[Livery]:
             logging.exception("Could not parse livery definition at %s", path)
             return None
         livery_name = data.get("name", path_id)
+
         countries_table = data.get("countries")
-        if countries_table is None:
-            countries = None
-        else:
-            countries = set(countries_table.values())
-        order = data.get("order", 0)
+        countries = None
```

Comment:
I can change this if you want, but I had left it with the same behavior as previous. I am surprised to see this change requested, as I figured that livery developers would take the 'lazy' way if they want a livery to be available to all countries.

Reply:
The prior behavior is that if the livery does not set the countries field, it is available for all countries. That is DCS's behavior, we have no say in it.

With your change applied, liveries with liveries fields that we failed to parse will be available to all countries in pydcs (because of the `except` branch does not `return None`). They probably are not available to all countries in DCS, because if that's what the author wanted, they wouldn't have set it at all.

Reply:
I'm betting I wasn't clear enough on which part I think the problem is. `countries = None` isn't actually the problem here, it's just the beginning of the changed block that has the problem, and the problem is a _missing_ line, which I of course can't comment on :) Adding `return None` after `logging.exception()` would work fine.

Reply:
If my explanation doesn't match your understanding, lmk which part is wrong. It seems like you and I are 100% agreed on the goal here, one of us has just misunderstood some part of the behavior either before or after.